easybuild/easyconfigs/r/R/ff_aarch64_support.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/utk_platform_macros.hpp b/src/utk_platform_macros.hpp
```

Comment:
@terjekv Please include a comment to describe the patch + mention author. If there are any relevant upstream issues/PRs, please mention them too.

Reply:
Added comments and upstream reference to the patch. Added author. (Me).